launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -472,6 +475,13 @@ void CSettingsView::on_buttonShowIntro_toggled(bool value)
 	updateCheckbuttonText(ui->buttonShowIntro);
 }
 
+void CSettingsView::on_buttonAllowPortrait_toggled(bool value)
+{
+	Settings node = settings.write["video"]["allowPortrait"];
+	node->Bool() = value;
+	updateCheckbuttonText(ui->buttonAllowPortrait);
+}
+
```

Comment:
Is there any point in making this option player-selectable, since there is no usable layout for vertical screen?